examples/Makefile.am

```diff
@@ -32,7 +27,9 @@ pkginclude_HEADERS =
 #
 # CPP TESTS
 #
+antioch_initdir = $(prefix)/share/examples/antioch_init
 antioch_init_SOURCES = antioch_init.C
+antioch_init_DATA    = ${antioch_init_SOURCES}
 
 #Define examples to actually be run
 TESTS  =
```

Comment:
Leaving them.  More tests is better than fewer, even if we're pretty sure they're redundant.  We probably don't want to establish the *expectation* that examples also be regression tests, but we should leave open the *option*.